functions.php

```diff
@@ -41,16 +41,29 @@ function as_enqueue_async_action( $hook, $args = array(), $group = '', $unique =
 		return is_int( $pre ) ? $pre : 0;
 	}
 
-	return ActionScheduler::factory()->create(
-		array(
-			'type'      => 'async',
-			'hook'      => $hook,
-			'arguments' => $args,
-			'group'     => $group,
-			'unique'    => $unique,
-			'priority'  => $priority,
-		)
-	);
+	try {
+		return ActionScheduler::factory()->create(
+			array(
+				'type'      => 'async',
+				'hook'      => $hook,
+				'arguments' => $args,
+				'group'     => $group,
+				'unique'    => $unique,
+				'priority'  => $priority,
+			)
+		);
+	} catch ( Exception $exception ) {
+		ActionScheduler::logger()->log(
+			0,
```

Comment:
By logging to action zero, the logged message won't be surfaced anywhere (that I'm aware of).

- Should we support this with a change to the admin UI, to make it easier for site operators to find the problem?
- Or, as a minimum, support this change with notes in our docs?
- Or, perhaps consider using [error_log()](https://www.php.net/manual/en/function.error-log.php) (on the basis that the logged errors would be more findable).

Reply:
Great catch. For now I'll go with 2 and 3, but longer term I think 1 would also be desirable.

Reply:
Opened an issue for 1 here: #1002.